elements/pf-panel/pf-panel.css

```diff
@@ -44,19 +44,22 @@
   pointer-events: none;
   content: "";
   border:
-    var(--pf-c-panel--before--BorderWidth,
-      var(--pf-c-panel--m-bordered--before--BorderWidth,
-        var(--pf-global--BorderWidth--sm, 1px)))
+  var(--pf-c-panel--m-bordered--before--BorderWidth)
```

Comment:
please revert this line so that the fallback values are in place in case a user loads pf-panel without the global patternfly stylesheet